test/libyul/yulOptimizerTests/fullInliner/not_inside_for.yul

```diff
@@ -13,12 +14,12 @@
 // ----
 // {
 //     {
+        let a_3 := 0
```

Comment:
I'm surprised that this works, since it's missing the ``//`` at the line beginning - it seems they are in fact optional, but you should still add them for consistency. The same applies to all the other test expectations (everything below the ``// ====`` should start with ``//``.

Reply:
Done.